core/components/seosuite/elements/plugins/seosuite.plugin.php

```diff
@@ -2,6 +2,4 @@
 use Sterc\SeoSuite\SeoSuite;
 
 $seosuite = $modx->services->get('seosuite');
-if ($seosuite instanceof SeoSuite) {
-    $seosuite->firePlugins($modx->event, $scriptProperties);
-}
+$seosuite->firePlugins($modx->event, $scriptProperties);
```

Comment:
`modX::services->get` will throw an exception in case the service is not registered